### PR TITLE
Material API what's new

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
### Purpose of this PR

add section in what's new for #5761 

Also add what's new for main light node #5086

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
@adrien-de-tocqueville guess you will need to create a Jira task?

File: com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md [hunk 1]
```diff
@@ -4,4 +4,15 @@ This page contains an overview of new features, improvements, and issues resolve
...
+
+To enable or disable most HDRP shader features on a material, changes need to be done on the keyword state and sometimes on one or more properties. This is done automatically when editing values though the inspector, but from HDRP 13.0, new APIs are also available in order to run the validation steps from script, both in the editor and in standalone builds.
+
+For more information, see the [Material Scripting API documentation](Material-API.md).
+
+### Access main directional light from ShaderGraph
+
+From HDRP 13, you can access the main light direction from a ShaderGraph using the *Main Light Direction* node.
+For more information, see the [node documentation](https://docs.unity3d.com/Packages/com.unity.shadergraph@13.1/manual/Main-Light-Direction-Node.html).
+
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md:17]: ### New Main Light Direction ShaderGraph node

From HDRP version 13.0,  [ShaderGraph](https://docs.unity3d.com/Packages/com.unity.shadergraph@13.1/manual/index.html) includes a new node called **Main Light Direction** that you can use to control the direction of the main light.
For more information, see [Main Light Direction Node](https://docs.unity3d.com/Packages/com.unity.shadergraph@13.1/manual/Main-Light-Direction-Node.html).

File: com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md [hunk 2]
```diff
@@ -4,4 +4,15 @@ This page contains an overview of new features, improvements, and issues resolve
 
 ## Added
 
+### Material Runtime API
+
+To enable or disable most HDRP shader features on a material, changes need to be done on the keyword state and sometimes on one or more properties. This is done automatically when editing values though the inspector, but from HDRP 13.0, new APIs are also available in order to run the validation steps from script, both in the editor and in standalone builds.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md:9]: From this HDRP version, you can use new APIs to run shader validation steps from script at runtime, both in the editor and in standalone builds. You can use this to change the keyword state or one or more properties in order to enable or disable HDRP shader features on a Material.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Added re-write suggestions in comments. Changed the order of information to make the new feature more clear :)

**Comment by adrien-de-tocqueville** [comment]
i forgot to add the new node to the table of content for shadergraph doc. So i did it in this PR

**Comment by sebastienlagarde** [comment]
Feedback adressed, merging

**Pull request merged**